localization/gyro_odometer/launch/gyro_odometer.launch.xml

```diff
@@ -11,7 +11,7 @@
 
   <arg name="config_file" default="$(find-pkg-share gyro_odometer)/config/gyro_odometer.param.yaml"/>
 
-  <node pkg="gyro_odometer" exec="gyro_odometer" name="gyro_odometer" output="screen">
+  <node pkg="gyro_odometer" exec="gyro_odometer_node" output="screen">
```

Comment:
To save a message to a file if glog makes an error, clould you change `output=“screen”` to `output=“both”`?

Reply:
Thank you. I've done it.